lib/ui/views/main/main_view.dart

```diff
@@ -18,40 +18,45 @@ import 'main_viewmodel.dart';
 // it's just a UI representation.
 
 class MainView extends StatelessWidget {
-  const MainView({Key? key}) : super(key: key);
+  // ignore: prefer_const_constructors_in_immutables
+  MainView({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     final customTheme = Theme.of(context).extension<CustomTheme>()!;
 
     const int defaultViewIndex = 1;
+    MainViewModel mainViewModel = MainViewModel();
```

Comment:
It's not necessary to have another instance of `MainViewModel`. You can just reuse the `model` found in Line 36 below. `builder: (context, model, child) {...`

Reply:
ops i will look into this and fix it! sorry